generate_report.py

```diff
@@ -1,20 +1,268 @@
-#!/usr/bin/python
-#
-# This file is part of ARM's analytics reporting which is released under the Apache 2 license.
-# See file /LICENSE or go to https://github.com/ARM-open/analytics-reports arm for full license details.
-# 
+"""
+---------------------------------------------------------------------------------
+Takes user sessions off Google Analytics API and Puts a bar graph into a PDF File
+Uses Python3 Written by Arian Moslem
+---------------------------------------------------------------------------------
+"""
 
+import argparse
+import os
+import sys
+import subprocess
+
+from apiclient.discovery import build
+import httplib2
+from oauth2client import client
+from oauth2client import file
+from oauth2client import tools
+
+from jinja2 import Environment, FileSystemLoader
 
 import click
```

Comment:
another style thing, but move all the single package imports to the top of the file. 

So put import click on top.